pkg/cluster/config/default.go

```diff
@@ -26,8 +26,8 @@ func addDefaultingFuncs(scheme *runtime.Scheme) error {
 	return RegisterDefaults(scheme)
 }
 
-// SetDefaults_Config sets uninitialized fields to their default value.
-func SetDefaults_Config(obj *Config) {
+// SetDefaults_Cluster sets uninitialized fields to their default value.
+func SetDefaults_Cluster(obj *Cluster) {
```

Comment:
🤔 I didn't think defaults were usually applied as part of the internal API version? Although I could definitely be wrong 😄 (/me goes away to dig into Kubernetes)

Reply:
We have to default the internal type as well currently because its used by library consumers 🤔